src/style.css

```diff
@@ -3,32 +3,62 @@
 }
 
 .container {
-  max-width: 30%;
+  max-width: 40%;
   box-shadow: 0 2px 8px rgb(0 0 0 / 25%);
   margin: 50px;
   border: 1px solid whitesmoke;
   align-self: center;
 }
```

Comment:
- [ **OPTIONAL** ] What do you think to center the tasks' container to make your page looks smart. You can only do it by adding this value(`auto`) at the `margin` property i.e it looks like this:
```
.container {
  ....
  margin: 50px auto;
  ....
}
```